cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts

```diff
@@ -155,6 +156,79 @@ describe(["@tier3"], "Applications interlinked to tags and business service", ()
         cy.get(continueButton).click();
     });
 
+    it("Validates association application tags to  archetype tags ", function () {
+        //automates polarion MTA-401
+        //create archytype
+        tagList = createMultipleTags(3);
+        const archetype = new Archetype(
+            data.getRandomWord(8),
+            [tagList[0].name, tagList[2].name],
+            [tagList[1].name],
+            null
+        );
+        archetype.create();
+        cy.wait(2 * SEC);
+
+        //Create 3 appdatas containing different tags
+        let appdata1 = {
+            name: data.getAppName(),
+            description: data.getDescription(),
+            tags: [tagList[0].name, tagList[2].name],
+            comment: data.getDescription(),
+        };
+        let appdata2 = {
+            name: data.getAppName(),
+            description: data.getDescription(),
+            tags: [tagList[0].name, tagList[1].name],
+            comment: data.getDescription(),
+        };
+        let appdata3 = {
+            name: data.getAppName(),
+            description: data.getDescription(),
+            tags: [tagList[0].name, tagList[2].name],
+            comment: data.getDescription(),
+        };
```

Comment:
Don't do like this. Do a `for` cycle and push it each time to array, something like
```
let appDataList = [];
for (let i = 0; i < 3; i++) {
  appDataList.push({
   name: data.getAppName(),
   description: data.getDescription(),
   tags: [tagList[0].name, tagList[2].name],
   comment: data.getDescription(),
   };)
}
```
And move it to `beforeAll` section, if possible

Reply:
thanks for the review! due to the nature of the test case i changed it to using a map and for each iterative approach, unable to move it to beforeAll because i am creating unique combinations while other test cases are using other combinations :+1: